### DISCUSSION TITLE
Picker style loss,Can someone help me

### DISCUSSION THREAD

**Discussion by lvdougao1** [adobe/react-spectrum#6120] [Q&A]
photo shop 25.6.0
![image](https://github.com/adobe/react-spectrum/assets/89889954/08321580-f001-4716-a8a7-9b8b24e6bce9)
![image](https://github.com/adobe/react-spectrum/assets/89889954/06e3cf68-e9cb-434e-a1fb-d6bd94561f90)

**Comment by lvdougao1** [comment]
![image](https://github.com/adobe/react-spectrum/assets/89889954/525aa8d2-ffc6-4b7f-88e3-d716bd292109)

**Reply by snowystinger** [reply]
Have you checked https://github.com/adobe/react-spectrum/wiki/Frequently-Asked-Questions-(FAQs)#why-are-there-errors-after-upgrading-a-react-spectrum-package ?
Otherwise, we're going to need to see this in a codesandbox/stackblitz to know more about what is going on.

**Reply by lvdougao1** [reply]
This works fine in a web page, but in a photo shop it will cause an error, The codesandbox address is below

**Reply by lvdougao1** [reply]
https://codesandbox.io/p/github/lvdougao1/uxp-react-swc/main?layout=%257B%2522sidebarPanel%2522%253A%2522EXPLORER%2522%252C%2522rootPanelGroup%2522%253A%257B%2522direction%2522%253A%2522horizontal%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522id%2522%253A%2522ROOT_LAYOUT%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522clucbufy50006356hxsar75hu%2522%252C%2522sizes%2522%253A%255B60%252C40%255D%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522EDITOR%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522id%2522%253A%2522clucbufy50002356hf62jci9w%2522%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522SHELLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522id%2522%253A%2522clucbufy50004356h4rtz63l1%2522%257D%255D%252C%2522sizes%2522%253A%255B100%255D%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522DEVTOOLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522id%2522%253A%2522clucbufy50005356hp2bbn7dg%2522%257D%255D%252C%2522sizes%2522%253A%255B100%255D%257D%255D%252C%2522sizes%2522%253A%255B50%252C50%255D%257D%252C%2522tabbedPanels%2522%253A%257B%2522clucbufy50002356hf62jci9w%2522%253A%257B%2522id%2522%253A%2522clucbufy50002356hf62jci9w%2522%252C%2522tabs%2522%253A%255B%255D%257D%252C%2522clucbufy50005356hp2bbn7dg%2522%253A%257B%2522id%2522%253A%2522clucbufy50005356hp2bbn7dg%2522%252C%2522activeTabId%2522%253A%2522clucc7cib00n1356h0dey00ao%2522%252C%2522tabs%2522%253A%255B%257B%2522type%2522%253A%2522TASK_PORT%2522%252C%2522taskId%2522%253A%2522start%2522%252C%2522port%2522%253A8080%252C%2522id%2522%253A%2522clucc7cib00n1356h0dey00ao%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522path%2522%253A%2522%252F%2522%257D%255D%257D%252C%2522clucbufy50004356h4rtz63l1%2522%253A%257B%2522id%2522%253A%2522clucbufy50004356h4rtz63l1%2522%252C%2522activeTabId%2522%253A%2522clucc85mu00t1356hk0n7jdw0%2522%252C%2522tabs%2522%253A%255B%257B%2522type%2522%253A%2522TASK_LOG%2522%252C%2522taskId%2522%253A%2522start%2522%252C%2522id%2522%253A%2522clucc794900kh356hpwtfzhby%2522%252C%2522mode%2522%253A%2522permanent%2522%257D%252C%257B%2522id%2522%253A%2522clucc85mu00t1356hk0n7jdw0%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522TERMINAL%2522%252C%2522shellId%2522%253A%2522clucc86jc000cdlddfhfwaat5%2522%257D%255D%257D%257D%252C%2522showDevtools%2522%253Atrue%252C%2522showShells%2522%253Atrue%252C%2522showSidebar%2522%253Atrue%252C%2522sidebarPanelSize%2522%253A15%257D

**Reply by snowystinger** [reply]
Unfortunately this means it's an issue with your application and we can't do too much without a reproduction. You'll need to do a fair amount of debugging. I did notice the name of your codesandbox is uxp and swc? Uxp is the runtime you're in, are you internal to Adobe? If so, please reach out in slack. And is swc spectrum web components? Or what does that stand for?

**Comment by lvdougao1** [comment]
This works fine in a web page, but in a photo shop it will cause an error,

**Comment by lvdougao1** [comment]
![image](https://github.com/adobe/react-spectrum/assets/89889954/dab5f03b-62d4-411d-bb07-c1087f9ce795)
